tests/test.py

```diff
@@ -9029,8 +9029,8 @@ def testClientIDRead(self):
             sa.select([self.test_table])
                 .where(self.test_table.c.id == sa.bindparam('id')), {'id': data['id']})
 
-        with self.assertRaises(UnicodeDecodeError):
-            row = self.executor2.execute_prepared_statement(query, args)[0]
+        with self.assertRaises(asyncpg.exceptions.SyntaxOrAccessError):
```

Comment:
as I remember, default behaviour should remain as is. And there is will be new one: return error. So we should leave previous tests as is and add new with configuration that returns error and check it